### PR TITLE
Various improvements (see desc.)

### LINKED ISSUE #81: Integrate database
Add dummy values and expand functionality

### PR DESCRIPTION
**Pull request by mccrosno**
Made several changes (seen below), specifically major changes to the admin panel and making it much easier to add new products/users, view their relationships (e.g., all of the seller's listed products or the seller of a specific product), as well as edit the data. Additionally, I added a few new values into the User and Product models to reference when sorting or filtering the product list or users.

Viewing/understanding the changes might be a bit difficult based off the code (specifically in admin.py), so I recommend downloading the updated file and viewing through the admin dashboard at [http://localhost:8000/admin/](url) and signing in with username admin and password admin.

- changed timezone in settings.py file
- added dummy users and products for testing
- updated admin dashboard with user and product info
- updates to user and product models

Closes #81 and #82

### CODE REVIEW

File: eMarket/app/tests.py
Comment (Newt-Tea) [inline eMarket/app/tests.py:1]: What is this function for, possibly remove.

File: eMarket/app/templates/app/logout.html [hunk 1]
```diff
@@ -6,7 +6,13 @@
...
 <body>
-  <h1>You have been logged out</h1>
-  <a href="{% url 'login' %}">Login again</a>
+  <h2>Logout</h2>
+    <p> You are now Logged out!</p>
+    {% comment %}TO-DO User is not deauthorized at this point {% endcomment %}
+  </form>
+
+  {% if error %}
+    <p style="color:red;">{{ error }}</p>
```
Comment (Newt-Tea) [inline eMarket/app/templates/app/logout.html:15]: Readd the link back to login

File: eMarket/app/templates/app/product_list.html [hunk 2]
```diff
@@ -5,27 +5,5 @@
...
-          <p>Price: ${{ product.price }}</p>
-          <a href="{% url 'product_detail' product.id %}">View Details</a>
-        </li>
-      {% endfor %}
-    </ul>
-  </main>
-</body>
\ No newline at end of file
+<body>This is where products will be listed.</body>
+</html>
```
Comment (Newt-Tea) [inline eMarket/app/templates/app/product_list.html]: Is the product display being handled elsewhere?

File: eMarket/app/urls.py [hunk 3]
```diff
@@ -2,15 +2,11 @@
...
-    path('product_detail/<int:product_id>/', views.product_detail, name='product_detail'),
-    path('register/', views.register, name='register'),
-    path('registration_success/', views.registration_success, name='registration_success'),
-    path('add_product/', views.add_product, name='add_product'),
-    path('shopping_cart/', views.shopping_cart, name='shopping_cart'),
-    path('payment_confirmed/', views.payment_confirmed, name='payment_confirmed'),
-    path('order_history/', views.order_history, name='order_history'),
-    path('logout/', views.logout_view, name='logout'),
-    
+  path('product_list/', views.product_list, name='product_list'),
```
Comment (Newt-Tea) [inline eMarket/app/urls.py]: Several Needed urls were removed

File: eMarket/app/views.py [hunk 4]
```diff
@@ -90,26 +78,4 @@ def add_product(request):
   else:
     form = ProductCreationForm()
   
-  return render(request, 'app/add_product.html', {'form': form})
-
-# Shopping Cart Page
-def shopping_cart(request):
```
Comment (Newt-Tea) [inline eMarket/app/views.py]: Why are these being removed they don't block previous pages from being functional

**Review by Newt-Tea — CHANGES_REQUESTED** [review]

File: eMarket/app/tests.py
Comment (mccrosno) [reply eMarket/app/tests.py:1]: It was for me to test dummy values in the database when I tried integrating it. I will remove sometime today when I can

File: eMarket/app/templates/app/product_list.html (hunk 2 above)
Comment (mccrosno) [reply eMarket/app/templates/app/product_list.html]: Fixed

File: eMarket/app/urls.py (hunk 3 above)
Comment (mccrosno) [reply eMarket/app/urls.py]: Fixed

File: eMarket/app/templates/app/logout.html (hunk 1 above)
Comment (mccrosno) [reply eMarket/app/templates/app/logout.html:15]: Fixed

**Review by Newt-Tea — APPROVED** [review]
All changes look good merging

**Pull request merged**